portfolio/src/main/webapp/script.js

```diff
@@ -223,6 +234,10 @@ function hasOnlyDigits(value) {
     return /^\d+$/.test(value);
 }
 
+function isAlphanumeric(value) {
+    return /^[0-9a-z]+$/.test(value);
```

Comment:
Do we need to consider about capital?

Reply:
An updated commit changed the regex to accept capital letters